e2e/e2e.go

```diff
@@ -74,21 +74,41 @@ func BuildDependenciesList(ctx context.Context) ([]string, error) {
 	return depsList, nil
 }
 
+// getConfig gets config file from GitHub and parses it into DevenvConfig
+func getConfig(ctx context.Context, serviceName string, gh *github.Client, configFileName string) (*DevenvConfig, error) {
+	r, _, err := gh.Repositories.DownloadContents(ctx, "getoutreach", serviceName, configFileName, nil)
+	l := log.With().Str("service", serviceName).Str("file", configFileName).Logger()
+	defer func() {
+		if err := r.Close(); err != nil {
+			l.Warn().Msg("Unable to close GH reader")
+		}
+	}()
+	if err != nil {
+		l.Debug().Msg("Unable to find file in GH")
+		return nil, err
+	}
+	var dc *DevenvConfig
```

Comment:
nit: Generally only decode into the non-pointer type and reference it in the return, it's entirely a style nitpick.

Reply:
Fixed